serializations.md

```diff
@@ -15,29 +15,49 @@ user API.
 
 ## Static size object serialization
 
-#### Block header
+There are two types of blocks:
 
-For block headers all fields sizes are statically known and can be
+* key block - does not contain a list of transactions, therefore, it is
+not divided into a header and a list of transactions;
+* micro block - contains a header and a list of transactions.
+
+#### Key block
```

Comment:
I know it was not introduced by this PR though it is strange that heading passes from `##` to `####` without `###`.